src/logics/LRALogic.cc

```diff
@@ -131,22 +133,32 @@ LRALogic::LRALogic() :
     sym_store.setInterpreted(sym_Real_ITE);
 }
 
-const SymRef LRALogic::get_sym_Num_TIMES () const {return sym_Real_TIMES;}
-const SymRef LRALogic::get_sym_Num_DIV () const {return sym_Real_DIV;}
-const SymRef LRALogic::get_sym_Num_MINUS () const {return sym_Real_MINUS;}
-const SymRef LRALogic::get_sym_Num_PLUS () const {return sym_Real_PLUS;}
-const SymRef LRALogic::get_sym_Num_NEG () const {return sym_Real_NEG;}
-const SymRef LRALogic::get_sym_Num_LEQ () const {return sym_Real_LEQ;}
-const SymRef LRALogic::get_sym_Num_GEQ () const {return sym_Real_GEQ;}
-const SymRef LRALogic::get_sym_Num_LT () const {return sym_Real_LT;}
-const SymRef LRALogic::get_sym_Num_GT () const {return sym_Real_GT;}
-const SymRef LRALogic::get_sym_Num_EQ () const {return sym_Real_EQ;}
-const SymRef LRALogic::get_sym_Num_ZERO () const {return sym_Real_ZERO;}
-const SymRef LRALogic::get_sym_Num_ONE () const {return sym_Real_ONE;}
-const SymRef LRALogic::get_sym_Num_ITE () const {return sym_Real_ITE;}
-const SRef LRALogic::get_sort_NUM () const {return sort_REAL;}
+PTRef LRALogic::mkRealDiv(const vec<PTRef>& args)
+{
+    SimplifyConstDiv simp(*this);
+    vec<PTRef> args_new;
+    SymRef s_new;
+    if (args.size() != 2) {
+        throw OsmtApiException("Division operation requries exactly 2 arguments");
```

Comment:
We could fix this as well to comply to the standard (` (/ Real Real Real :left-assoc)`)

Reply:
I am not sure I understand what that means. If I write `x / 2 / 3`, it should be read as `(x/2)/3`, right? This means we can accept more than two arguments, but each except the first one must be constant, right? Can we do it in another PR?

Reply:
Ok, another PR.